src/containers/TemplateBuilder/evaluatorGui/semanticComponentLibrary.tsx

```diff
@@ -32,13 +32,14 @@ const ComponentLibrary: ComponentLibraryType = {
 
     return (
       <div className="io-wrapper">
-        <div className="io-component key">{title}</div>
+        {title && <div className="io-component key">{title}</div>}
         <Input
           value={innerValue}
+          // type="number"
```

Comment:
TO be removed please

Reply:
Can we leave this one in? It strictly *should* be active to let the native UI enforce number entry correctly, but Andrei made so many modifications to the behaviour of these components that by turning it on it was causing other issues. But I'd like to leave it there as a reminder that it should be set to `type=number` for a NumberInput component.